components/widgets/menu/menu_items/submenu_item.jsx

```diff
@@ -102,13 +102,39 @@ export default class SubMenuItem extends React.PureComponent {
             );
         }
 
+        const hasSubmenu = subMenu && subMenu.length;
         const parentWidth = this.node && this.node.current ? this.node.current.getBoundingClientRect().width - 19 : 0;
-
         const subMenuStyle = {
-            visibility: this.state.show && subMenu && subMenu.length && !isMobile ? 'visible' : 'hidden',
+            visibility: this.state.show && hasSubmenu && !isMobile ? 'visible' : 'hidden',
             right: (parseInt(xOffset, 10) - 1) + 'px',
         };
 
+        let subMenuContent = '';
+
+        if (!isMobile) {
+            subMenuContent = (
+                <ul
+                    className={'a11y__popup Menu dropdown-menu SubMenu'}
+                    style={subMenuStyle}
+                >
+                    {isMobile || !subMenu ? '' : subMenu.map((s) => {
```

Comment:
This ìsMobile should be always false here, so you can remove it.

Reply:
Good catch, thank you.